src/blocks/lists/Feature/Component.client.tsx

```diff
@@ -0,0 +1,260 @@
+            }}
+            viewport={{
+              once: false,
+              amount: 0.2,
+            }}
+            whileInView={{
+              opacity: 1,
+              transition: {
+                duration: 0.6,
+                ease: 'easeOut',
+              },
+            }}
+          >
+            <div className="flex flex-col gap-5 group 2xl:items-end">
+              {features.map((feature, i) => (
+                <div
+                  key={i}
+                  className="flex gap-4 items-center cursor-pointer w-fit 2xl:flex-row-reverse"
+                  onClick={() => handleFeatureClick(feature.id || '')}
+                >
+                  <FeatureIcon
+                    height="24px"
+                    icon={feature.icon || 'banana'}
+                    iconClasses={`${
+                      currentSection === feature.id
+                        ? 'stroke-z-' + accentColor
+                        : 'stroke-[#5A5A5A]'
+                    }`}
+                    width="24px"
+                  />
+                  <p className="flex-1 whitespace-nowrap 2xl:transition-opacity 2xl:opacity-0 2xl:group-hover:opacity-100">
+                    {feature.featureName}
+                  </p>
+                </div>
+              ))}
+            </div>
+            <button className="cursor-pointer" onClick={scrollToBottom}>
+              <IconArrowDown color="black" height="24px" width="24px" />
+            </button>
+          </motion.div>
+        )}
+      </AnimatePresence>
+
+      {/* Container for feature sections */}
+      <div ref={containerRef} className="flex flex-col w-full">
+        {features.map((feature, i) => (
+          <div
+            key={i}
+            ref={(el) => {
+              featureSectionRefs.current[i] = el;
+            }}
+            className="flex flex-col -mx-5 md:mx-0 px-5 md:px-0 py-12 gap-12 border-t last:border-b bg-[#FCFCFC] md:bg-transparent group md:flex-row md:items-center md:justify-center overflow-hidden"
+            id={feature.id || ''}
+          >
+            <div className="flex flex-col gap-2.5 md:gap-5 md:flex-1 md:py-24">
+              <div className="flex flex-col gap-2.5 md:gap-3">
+                <div className="flex gap-2 items-center">
+                  <FeatureIcon
+                    color="#919191"
+                    height="20px"
+                    icon={feature.icon || 'banana'}
+                    width="20px"
+                  />
+                  <p className="text-[#646464] text-base font-light leading-[1.7]">
+                    {feature.featureName}
+                  </p>
+                </div>
+                <h4 className="text-2xl leading-[1.7] md:text-3xl font-medium">
+                  {feature.header}
+                </h4>
+              </div>
+              <p className="text-[#646464] text-lg font-light leading-[1.7]">
+                {feature.description}
+              </p>
+            </div>
```

Comment:
I'm missing the link, which I assume needs to have a user-editable text. Screenshot from figma below.

![image](https://github.com/user-attachments/assets/7480ddb8-ed8b-4adb-83b9-9861ea655395)